src/com/ECS/System.java

```diff
@@ -882,7 +992,23 @@ public static void LoadPicturesToEntity1(Entity entity,Image img){
 
     public void Process(ArrayList<Entity> entities, Graphics2D g) {
         for (Entity entity : entities) {
-            if (entity.hasComponent(CoinComponent.class) && entity.hasComponent(RenderComponent.class)) {
+            if (entity.hasComponent(PlatformComponent.class) && entity.hasComponent(RenderComponent.class)) {
```

Comment:
New rendering logic for Platform, so it can draw the platform depended on the platform width.